fe/fe-core/src/main/java/com/starrocks/common/proc/ComputeNodeProcDir.java

```diff
@@ -54,6 +55,7 @@ public class ComputeNodeProcDir implements ProcDirInterface {
                 .addAll(TITLE_NAMES)
                 .add("StarletPort")
                 .add("WorkerId")
+                .add("WarehouseName")
```

Comment:
‌‌If in share-nothing mode, what is the value of WarehouseName column?

Reply:
default_warehouse